lib/tasks/one_time.rake

```diff
@@ -0,0 +1,8 @@
+namespace :one_time do
+  desc "Populate totp_seed from mfa_seed"
+  task populate_totp_seed: :environment do
+    User.find_each do |user|
```

Comment:
nit: Could we also only update users that have a `mfa_seed` present instead of all of them? I believe that would save on the number of users that we need to process.